docs/deploy-cephfs.md

```diff
@@ -208,6 +208,7 @@ The same requirements from the Kubernetes section apply here, i.e. Kubernetes
 version, privileged flag and shared mounts.
 
 The Helm chart is located in `charts/ceph-csi-cephfs`.
+The default configmap name is `ceph-csi-config-cephfs` instead of `ceph-csi-config`.
```

Comment:
@chenxu1990 Thanks for the PR. we can rename the configmap name in helm charts to `ceph-csi-config` and update the upgrade documentation to edit/update the configmap from ceph-csi-config-cephfs` and `ceph-csi-config-rbd` to `ceph-csi-config` so the user will have one configmap for both cephfs and rbd.